gcp_variant_transforms/transforms/infer_headers_test.py

```diff
@@ -401,3 +417,141 @@ def test_pipeline(self):
       assert_that(inferred_headers,
                   asserts.header_fields_equal_ignore_order([expected]))
       p.run()
+
+  def test_infer_annotation_types_no_conflicts(self):
+    anno_fields = ['CSQ']
+    header = self._get_sample_header_fields_with_annotation()
+    variant = self._get_sample_variant_1()
+    variant.info['CSQ'] = ['A|GENE1|100|1.2', 'TT|GENE1|101|1.3']
+    infer_header_fields = infer_headers._InferHeaderFields(False, anno_fields)
```

Comment:
nit: please use `_InferHeaderFields(infer_headers=False, annotation_fields_to_infer=anno_fields)` so that it's more clear what the `False` stands for.